greenheart/tools/eco/finance.py

```diff
@@ -530,18 +530,18 @@ def run_opex(
 
     # solar opex
     if "pv" in hopp_config["technologies"].keys():
-        solar_opex = hopp_results["hybrid_plant"].pv.om_fixed + np.sum(
-            hopp_results["hybrid_plant"].pv.om_variable
-        )
+        solar_opex = hopp_results["hybrid_plant"].pv.om_total_expense[
+            0
+        ]
         if solar_opex < 0.1:
             raise (RuntimeWarning(f"Solar OPEX returned as {solar_opex}"))
     else:
         solar_opex = 0.0
 
     # battery opex
     if "battery" in hopp_config["technologies"].keys():
-        battery_opex = hopp_results["hybrid_plant"].battery.om_fixed + np.sum(
-            hopp_results["hybrid_plant"].battery.om_variable
+        battery_opex = (np.sum(hopp_results["hybrid_plant"].battery.om_fixed) 
+                        * hopp_config["technologies"]["battery"]["system_capacity_kw"]
```

Comment:
I think this should still use the om_total_expense for the battery as you did in the PV for GreenHEART, but we should set the om_variable to 0 for the battery if we don't want any variable expense.